packages/lib-webpack/src/webpack/DynamicRemotePlugin.ts

```diff
@@ -7,7 +7,7 @@ import type { Extension } from '@openshift/dynamic-plugin-sdk';
 import * as _ from 'lodash-es';
 import * as glob from 'glob';
 import * as path from 'path';
-import * as webpack from 'webpack';
+import webpack from 'webpack';
```

Comment:
This seems related to https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports

If we want to change this, we should update all such imports
```
$ git grep " from 'webpack';"
packages/lib-webpack/src/webpack/DynamicRemotePlugin.ts:import * as webpack from 'webpack';
packages/lib-webpack/src/webpack/GenerateManifestPlugin.ts:import * as webpack from 'webpack';
packages/lib-webpack/src/webpack/PatchContainerEntryPlugin.ts:import * as webpack from 'webpack';
```

What's important is how this gets reflected into `packages/lib-webpack/dist/index.js` output.

Reply:
Good point! updated.